gcp/appengine/frontend3/src/templates/home.html

```diff
@@ -196,13 +197,47 @@ <h3 class="code-card-title">Scan directory recursively</h3>
             <mwc-icon-button class="icon" icon="content_copy"></mwc-icon-button>
           </clipboard-copy>
         </div>
+        <div class="code-card mdc-layout-grid__cell--span-6 mdc-layout-grid__cell--span-12-tablet">
+          <h3 class="code-card-title">Guided Remediation (basic)</h3>
+          <pre class="code-card-content" id="example-guided-remediation-basic">
+osv-scanner fix --non-interactive --strategy=in-place -L path/to/package-lock.json
+osv-scanner fix --non-interactive --strategy=relock -M path/to/package.json -L path/to/package-lock.json
+          </pre>
+          <clipboard-copy class="code-card-copy" for="example-guided-remediation-basic">
+            <mwc-icon-button class="icon" icon="content_copy"></mwc-icon-button>
+          </clipboard-copy>
+        </div>
+        <div class="code-card mdc-layout-grid__cell--span-6 mdc-layout-grid__cell--span-12-tablet">
+          <h3 class="code-card-title">Guided Remediation (advanced)</h3>
+          <pre class="code-card-content" id="example-guided-remediation-advanced">
+osv-scanner fix -M path/to/package.json -L path/to/package-lock.json
```

Comment:
Can we link specifically to https://google.github.io/osv-scanner/experimental/guided-remediation/ for the guided remediation sections somehow? 

I think we can break out the "Command Line Tools" section into two to  make this work: "Vulnerability Scanner", and "Remediation Tools"